pkg/api/router.go

```diff
@@ -54,6 +54,10 @@ func (s *server) setupRouting() {
 		"POST": http.HandlerFunc(s.chunkUploadHandler),
 	})
 
+	handle(router, "/bzz:/{address}/{path:.*}", jsonhttp.MethodHandler{
```

Comment:
we agreed to use /bzz:/ notation?
https://beehive.ethswarm.org/swarm/pl/fzgc13trxpgjz8k37ft3y8zzgo

Reply:
https://beehive.ethswarm.org/swarm/pl/sfyot87eqin68cpsum3qspwp7w, I guess we just agreed to try `/bzz/`?